app/src/components/building-select-input.tsx

```diff
@@ -44,8 +49,8 @@ const BuildingTypeSelectInput: FC<BuildingTypeSelectInputProps> = ({
         borderRadius="4px"
         borderWidth={errors?.[activity] ? "1px" : 0}
         border="inputBox"
-        borderColor={errors?.[activity] ? "sentiment.negativeDefault" : ""}
-        background={errors?.[activity] ? "sentiment.negativeOverlay" : ""}
+        borderColor={error ? "sentiment.negativeDefault" : ""}
+        background={error ? "sentiment.negativeOverlay" : ""}
```

Comment:
Why switch to just error here? This would mean it's no longer specific to the activity you're in? If we need to make this change, also change it on the `borderWidth`, otherwise that could get inconsistent.